recipes/django-haystack/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "django-haystack" %}
+{% set version = "2.8.1" %}
+{% set sha256 = "8b54bcc926596765d0a3383d693bcdd76109c7abb6b2323b3984a39e3576028c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - django
```

Comment:
The `setup.py` says django >= 1.11. Also, it looks like this uses `pkg_resources`, so you'll need `setuptools as a run time dependency.